### PR TITLE
add better message when loading ref book exercises

### PR DESCRIPTION
**Pull request by philschatz**
Instead of saying "Loading..." it says "Loading Exercise...".

TODO:

- [x] get "Error Loading Exercise" to show up, ask UX about preferred message
- [x] decide what to do about missing exercises, ask UX

screen shot 2015-07-30 at 2 42 57 pm

Dependent on openstax/tutor-server#543

### CODE REVIEW

**Comment by philschatz** [comment]
@pandafulmanda Thoughts on this? I was trying to make the loading/error message clearer since a lot of people were getting confused when an exercise failed

**Comment by pandafulmanda** [comment]
Seems like a good idea, i will be updating exercises locally and checking how it looks.

File: src/components/book-content-mixin.cjsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 _ = require 'underscore'
 S = require '../helpers/string'
 
+EXERCISE_LINK_SELECTOR = '.os-exercise .autogenerated-content'
```
Comment (philschatz) [inline src/components/book-content-mixin.cjsx]: Why is this not `.os-exercise .os-embed`? (or `.os-exercise [data-type='problem'] > p > .os-embed`)? `.autogenerated-content` does not define the link as an exercise link

Comment (pandafulmanda) [inline src/components/book-content-mixin.cjsx]: I did not see .os-embed in there

**Pull request merged**